eatin-client/src/components/functions/useInsertNewUserRecipe.ts

```diff
@@ -1,29 +1,62 @@
 import { useCallback, useState } from "react";
-import { CreateUserRecipesMutation, useCreateUserRecipesMutation } from "../../generated/graphql";
+import {
+    CreateUserRecipesMutation,
+    useCreateUserRecipesMutation,
+    useGetUserrecipesByRecipeAndUserQuery,
+} from "../../generated/graphql";
 import { useGetUsersName } from "../hooks/useGetUsersName";
 
-export function useInsertNewUserRecipe() {
+export function useInsertNewUserRecipe(recipeIndex: number) {
     const [createSavedUserRecipe] = useCreateUserRecipesMutation();
     const [data, setData] = useState<CreateUserRecipesMutation | null | undefined>(null);
     const userID = useGetUsersName();
+    const { data: recipeConnection } = useGetUserrecipesByRecipeAndUserQuery({
```

Comment:
```suggestion
    const { data: recipeConnection } = useGetUserRecipesByRecipeAndUserQuery({
```

Reply:
the CamelCase in the name is not in all the places it should

Reply:
Its generated like that by graphql